.gitlab/e2e.yml

```diff
@@ -262,6 +262,14 @@ new-e2e-cws-dev:
   # Temporary, until we manage to stabilize those tests.
   allow_failure: true
 
+new-e2e-cws-main:
+  extends: .new_e2e_template
+  rules: !reference [.on_main_and_no_skip_e2e]
+  variables:
+    TARGETS: ./tests/cws
+  # Temporary, until we manage to stabilize those tests.
+  allow_failure: true
+
 #   ^    If you create a new job here that extends `.new_e2e_template`,
 #  /!\   do not forget to add it in the `dependencies` statement of the
 # /___\  `e2e_test_junit_upload` job in the `.gitlab/e2e_test_junit_upload.yml` file
```

Comment:
As specified in the comment, it would be good to have this new job added in the dependencies of the `e2e_test_junit_upload` job of the `.gitlab/e2e_test_junit_upload.yml` file in order to have the test results available in the [datadog CI app](https://app.datadoghq.com/ci/test-runs?query=test_level%3Atest%20%40test.service%3Adatadog-agent%20%40git.branch%3Amain&citest_explorer_sort=time%2Cdesc&index=citest&mode=sliding&saved-view-id=2236559&start=1700641464310&end=1701246264310&paused=false).

Reply:
Good point, thanks for reminding me. This should be good in the last commit